CHANGELOG.md

```diff
@@ -6,7 +6,10 @@ and the versioning aims to respect [Semantic Versioning](http://semver.org/spec/
 ## [1.X.X] - 20XX-XX-XX
 
 ### Added
+- added built ontology to the pipeline artifacts.
 ### Changed
+- changed order in which the oeo-full files are compiled, owl now builds before omn.
+- updated part relation between electricity grid and power line to have simple restrictions.
```

Comment:
Here only the class `electricity grid` should be mentioned, without the details of the changes.